### ISSUE TITLE
Callback registration

### ISSUE THREAD

**Issue by sschw** [z3r0privacy/EasyKeepassBrowserPlugin#1]
Dont you want to start this when the page has finished loading its content?
This would look like the following code

```
var callback = function(){
  pluginAbc123.FindAndFillLoginFields(null); 
};

if (
    document.readyState === "complete" ||
    (document.readyState !== "loading" && !document.documentElement.doScroll)
) {
  callback();
} else {
  document.addEventListener("DOMContentLoaded", callback);
}
```
https://github.com/z3r0privacy/EasyKeepassBrowserPlugin/blob/136d58cec35b72532605255904c56b42af2c193e/AddOn/ekbp.js#L179-L181

**Comment by z3r0privacy** [comment]
Why do you need `(!document.documentElement.doScroll)`?

 --> included in 37ecaddca264d03077028412a0852f201635f912, see https://github.com/z3r0privacy/EasyKeepassBrowserPlugin/blob/37ecaddca264d03077028412a0852f201635f912/AddOn/content/ekbp.js#L226-L233

**Comment by sschw** [comment]
idk its the same way as JQuery does `$.ready(...)`
